docs/latest/tutorial/tutorial-scaffolding.md

```diff
@@ -0,0 +1,354 @@
+instance with a series of options:
+
+```js
+const createWindow = () => {
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      sandbox: true
+    }
+  })
+
+  win.loadFile('index.html')
+}
+```
+
+:::tip
+You can find all the options available in the [`BrowserWindow` page][browser-window].
+:::
+
+In Electron, browser windows can only be created after the `app` module's
+[`ready`][app-ready] event is fired. You can wait for this event by using the
+[`app.whenReady()`][app-when-ready] API. Call `createWindow()` after `whenReady()`
+resolves its Promise. This is done in the last 3 lines of the code:
+
+```js
+app.whenReady().then(() => {
+  createWindow()
+})
+```
+
+At this point, your Electron application should successfully open a window that displays your
+web page!
+
+You have now code being executed in the main and renderer process.
+The easiest way to debug the code in your renderer process is by using the built-in developer
+tools. You can invoke them with a shortcut (`Ctrl/Command+Shift+I`) or programmatically
+via `mainWindow.webContents.openDevTools()` in the main process. If you want to use VS Code,
+keep reading!
+
+The renderer has access to the web APIs. That means you can use the same JavaScript APIs and
+tooling you use for typical front-end development, such as using [`webpack`][webpack] to bundle
+and minify your code or [React][react] to manage your user interfaces.
+
+:::tip
+You can find more information about this subject in the [Boilerplates and CLIs].
+:::
+
+### Debugging from VS Code
+
+If you want to debug your application using VS Code instead of using the terminal you have to
+keep in mind there are 2 processes to attach at this moment: the main and renderer processes.
+
+We are going to create a configuration file that allow us to debug both at the same time:
+
+1. Create a new folder `.vscode` in the root of your project.
+1. Create a new file `launch.json` with the following content:
+
+```json title='.vscode/launch.json'
+{
+  "version": "0.2.0",
+  "compounds": [
+    {
+      "name": "Main + renderer",
+      "configurations": ["Main", "Renderer"],
+      "stopAll": true
+    }
+  ],
+  "configurations": [
+    {
+      "name": "Renderer",
+      "port": 9222,
+      "request": "attach",
+      "type": "pwa-chrome",
+      "webRoot": "${workspaceFolder}"
+    },
+    {
+      "name": "Main",
+      "type": "pwa-node",
+      "request": "launch",
+      "cwd": "${workspaceFolder}",
+      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron",
+      "windows": {
+        "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron.cmd"
+      },
+      "args": [".", "--remote-debugging-port=9222"],
+      "outputCapture": "std",
+      "console": "integratedTerminal"
+    }
+  ]
+}
+```
+
+<!-- markdownlint-disable MD029 -->
+
+3. The option `Main + renderer` will appear when you select `Run and Debug`
```

Comment:
```suggestion
1. The option `Main + renderer` will appear when you select `Run and Debug`
```

Reply:
It has to be a 3, otherwise the numbering starts again at 1. That's why I have disabled the markdownlint rule

Reply:
Oh I see! I missed that.